src/cli/Cli.ts

```diff
@@ -404,17 +404,68 @@ export class Cli {
       const commandTypes = commandInfo.command.types;
       if (commandTypes) {
         minimistOptions.string = commandTypes.string;
-        minimistOptions.boolean = commandTypes.boolean;
+
+        // Only treat those options as booleans that are included as arg
+        // Otherwise unincluded options will default to false.
+        minimistOptions.boolean = commandTypes.boolean?.filter(optionName => args.some(arg => `--${optionName}` === arg || `-${optionName}` === arg));
```

Comment:
Since we're iterating over args that were specified rather than all options defined as bool, do we need this line?

Reply:
Good point... if we leave this in, minimist will convert 'asdasdads' to 'true'. At least, I'm seeing that behavior. 
If we leave this out, minimist parses the arguments as string values. We'd need to validate them anyway and type them as strings.

Do you see what I mean?

Alternative solution is that we could also throw an error if the value is not truthy or falsy. Combined with keeping the line in, we get booleans instead of strings into the code.

Reply:
Throwing validation exception would be the best way about it. It's a shame we can't make it a part of our validation logic and need to do it earlier but I don't see a better way about it other than defining everything as a string, which would likely come with its own set of side effects.

Reply:
`Error: The value 'sfdsfdfs' for option '--allowCalendarSharing' is not a valid boolean`

Reply:
@waldekmastykarz :

![image](https://user-images.githubusercontent.com/5267487/199609585-64aee938-865b-40bc-9533-e0f9b5be9f86.png)